cmd/clusterctl/cmd/delete.go

```diff
@@ -109,6 +110,8 @@ func init() {
 		"IPAM providers and versions (e.g. infoblox:v0.0.1) to delete from the management cluster")
 	deleteCmd.Flags().StringSliceVar(&dd.runtimeExtensionProviders, "runtime-extension", nil,
 		"Runtime extension providers and versions (e.g. test:v0.0.1) to delete from the management cluster")
+	deleteCmd.Flags().StringSliceVar(&dd.addonProviders, "addon", nil,
+		"Add-on providers and versions (e.g. helm:v0.1.0) to delete from the management cluster")
```

Comment:
```suggestion
		"Addon providers and versions (e.g. helm:v0.1.0) to delete from the management cluster")
```

Reply:
Similar in other files in this package

Reply:
I've been leaving it spelled as "add-on" in docs/text since that's how it's been spelled in the proposal doc. I think we've been using "addon" in variable or file names so far.

Reply:
Interesting. I would have assumed that we spell it either "add-on" or "addon" which should then result in either "AddOn" or "Addon" variable names.

Reply:
I won't block the PR because of the godoc, but I assume we have consensus that in other places (including the command line flags) we want to call it "addon"?
(cc @ykakarap @fabriziopandini)